web/rainmaker/dev-packages/egov-ui-kit-dev/src/common/common/Header/index.js

```diff
@@ -43,6 +43,17 @@ class Header extends Component {
     updateActiveRoute(menupath, menuName);
   };
 
+  componentWillReceiveProps = (nextProps) => {
+    const { role, userInfo } = this.props;
+    const permanentCity = get(nextProps, "userInfo.permanentCity");
+    if (get(userInfo ,"permanentCity") !== get(nextProps, "userInfo.permanentCity")) {
+      const tenantId = role.toLowerCase() === "citizen" ? permanentCity : getTenantId();
+      const ulbLogo = `https://s3.ap-south-1.amazonaws.com/pb-egov-assets/${tenantId}/logo.png`;
```

Comment:
This cannot be hardcoded, should come from MDMS. Please don't hardcode anything specific to Punjab